pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dto/kit/ClinicalKitDto.java

```diff
@@ -101,25 +101,25 @@ public void setGender(String genderString) {
     }
 
     public void setNecessaryParticipantDataToClinicalKit(String ddpParticipantId, DDPInstance ddpInstance) {
+        logger.info("Setting PHI to clinical kit's accessioning data for {}", ddpParticipantId);
         Optional<ElasticSearchParticipantDto> maybeParticipantESDataByParticipantId =
                 ElasticSearchUtil.getParticipantESDataByParticipantId(ddpInstance.getParticipantIndexES(), ddpParticipantId);
         if (maybeParticipantESDataByParticipantId.isEmpty()) {
             throw new RuntimeException("Participant ES Data is not found for " + ddpParticipantId);
         }
-
+        ElasticSearchParticipantDto elasticSearchParticipantDto = maybeParticipantESDataByParticipantId.get();
         try {
-            this.setDateOfBirth(maybeParticipantESDataByParticipantId.get().getDsm().map(Dsm::getDateOfBirth).orElse(""));
-            this.setFirstName(maybeParticipantESDataByParticipantId.get().getProfile().map(Profile::getFirstName).orElse(""));
-            this.setLastName(maybeParticipantESDataByParticipantId.get().getProfile().map(Profile::getLastName).orElse(""));
-            this.setGender(getParticipantGender(maybeParticipantESDataByParticipantId.get(), ddpInstance.getName()));
-            String shortId = maybeParticipantESDataByParticipantId.get().getProfile().map(Profile::getHruid).orElse("");
+            this.setDateOfBirth(elasticSearchParticipantDto.getDsm().map(Dsm::getDateOfBirth).orElse(""));
+            this.setFirstName(elasticSearchParticipantDto.getProfile().map(Profile::getFirstName).orElse(""));
```

Comment:
This code should first establish whether `profile` and `dsm` sections exist in the participant document, and handle those situations separately. (Do we want to proceed is those cases or throw an error?)

Reply:
Unless I am mistaken that should be handled by `orElse` . Am I wrong?

Reply:
Yup, but I was wondering if we want to proceed if either DSM or profile are empty, but get that's probably out of scope for this PR.

Reply:
Yeah not sure if that's an error or an empty string should be returned in that case